plotter/scatterColor_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plotter
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"testing"
+
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/internal/cmpimg"
+	"gonum.org/v1/plot/palette/moreland"
+	"gonum.org/v1/plot/vg"
+	"gonum.org/v1/plot/vg/draw"
+	"gonum.org/v1/plot/vg/vgimg"
+)
+
+// ExampleScatterColor draws some scatter points, a line,
```

Comment:
please rename this as: `ExampleScatter_color` so it looks well on godoc (and is part of the examples for `Scatter`.)

Reply:
still there :)
also, the comment doesn't match what the example actually does :)

perhaps something along these lines would do ?
```go
// ExampleScatter_color draws some scatter points.
// Each point will be plotted with a different color depending on some external criteria.
func ExampleScatter_color() {
```

(it could be improved, though.)